### DISCUSSION TITLE
GTX 1080 Ti

### DISCUSSION THREAD

**Discussion by YouriAndropov** [LeelaChessZero/lc0#1716] [Q&A] [answered]
Hello,

How to figure what net is the most appropriate for my GTX 1080 Ti using cudnn ?
My preliminary tests seem to favor 24x320 and 30x384 over 20x256.
I'm still conducting tests but did anyone also try ?

**Comment by Naphthalin** [comment, marked as answer]
In general, for this question we maintain lc0.org/bestnet though it still misses the most recent nets.

I recommend trying the cuda (not cudnn) v0.29.0 with the recent T78/T80 nets and see for yourself.
